packages/pg-pubsub/src/index.ts

```diff
@@ -161,6 +161,7 @@ const plugin: PostGraphilePlugin = {
       let client: pg.PoolClient;
       try {
         client = await pgPool.connect();
+        client.on("error", () => {});
```

Comment:
We need to remove this listener in `cleanClient`:

https://github.com/graphile/graphile-engine/blob/b7db0ba692ab8885a5356098dc59bb90dd5b9088/packages/pg-pubsub/src/index.ts#L88

Create a `function noop(){}` somewhere so that we can add it by reference:

```suggestion
        client.on("error", noop);
```

Reply:
https://github.com/graphile/graphile-engine/pull/791/commits/f25b47c50ba189eedab0708242d7d2415a45db97